tusd/hooks/implementations/hooks_utils.py

```diff
@@ -42,13 +42,13 @@ def get_data_from_db(conn, invitasjon_ekstern_id: str, logger) -> Optional[DataF
     try:
         dict_cursor = conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
         dict_cursor.execute(
-            'SELECT i.ekstern_id    AS ekstern_id, '
+            'SELECT i.id            AS invitasjon_id'
+            'i.ekstern_id           AS ekstern_id, '
```

Comment:
```suggestion
            'SELECT i.id            AS invitasjon_id, '
            'i.ekstern_id           AS ekstern_id, '
```
Mangler komma, gir denne feilen `LINE 1: SELECT i.id            AS invitasjon_idi.ekstern_id         ...`